config/application.yml

```diff
@@ -13,6 +13,7 @@ default: &default
 
 development:
   <<: *default
+  DEVISE_SECRET_KEY: "6a8b67c33f0cd9be1a41ab9989bcceeb1cc46916dc494b105092513efb8c7d2e3c5fc04bd88b6fc0163c0dcbc141a5e65effbfe78fc65090a5cf4fcaf06179b9"
```

Comment:
it's ok to have the same key for both development and test @Goose97 so I prefer to move this into the `default` scope above :)

Reply:
Fixed in 8b9677d461a7193c07c15494b777769217975d10